network-api/networkapi/wagtailpages/views.py

```diff
@@ -22,3 +24,11 @@ def custom404_view(request, exception):
     else:
         html = render(request, '404.html')
         return HttpResponseNotFound(html.content)
+
+
+def redirect_to_initiatives(request, rest):
+    lang = request.LANGUAGE_CODE
+    translation.activate(lang)
+    request.session[translation.LANGUAGE_SESSION_KEY] = lang
+
+    return redirect(urljoin('/', 'initiatives/' + rest))
```

Comment:
do we need `urljoin` here? Can we use `f'/initiates/{rest}'`?

Reply:
`f'/initiatives/{rest}'` works. PR has been updated.